cmd/oci-image-tool/man/oci-image-tool-create-runtime-bundle.1.md

```diff
@@ -2,14 +2,20 @@
 % OCI Community
 % JULY 2016
 # NAME
-oci-image-tool-create-runtime-bundle \- Create an OCI image runtime bundle
+oci-image-tool-create-runtime-bundle \- Create an OCI runtime bundle
 
 # SYNOPSIS
 **oci-image-tool create-runtime-bundle** [src] [dest] [flags]
 
 # DESCRIPTION
 `oci-image-tool create-runtime-bundle` generates an [OCI bundle](https://github.com/opencontainers/runtime-spec/blob/master/bundle.md) from an [OCI image layout](https://github.com/opencontainers/image-spec/blob/master/image-layout.md).
```

Comment:
I'm not sure how we intend to compile these to troff, but ocitools uses [go-md2man][1], which currently handles links poorly:

    $ go-md2man -in oci-image-tool-create-runtime-bundle.1.md -out oci-image-tool-create-runtime-bundle.1
    $ grep -A4 DESCRIPTION oci-image-tool-create-runtime-bundle.1
    .SH DESCRIPTION
    .PP
    \fB\fCoci\-image\-tool create\-runtime\-bundle\fR generates an 
    \[la]https://github.com/opencontainers/runtime-spec/blob/master/bundle.md\[ra] from an 
    \[la]https://github.com/opencontainers/image-spec/blob/master/image-layout.md\[ra]\&.

So I recommend we drop this line now that we're adding the additional detail below.  Just start with:

    `oci-image-tool create-runtime-bundle` validates an application/vnd.oci.image.manifest.v1+json…


[1]: https://github.com/cpuguy83/go-md2man